src/components/atoms/textInput/text-input.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryFn } from '@storybook/react'
+
+import type { TextInputProps } from 'typings/components'
+import { TextInput } from 'components'
+
+export default {
+  title: 'Atoms/TextInput',
+  component: TextInput,
+  argTypes: {
+    label: {
+      control: {
+        type: 'text',
+      },
+    },
+    message: {
+      control: {
+        type: 'text',
+      },
+    },
+    messageType: {
```

Comment:
```suggestion
    status: {
```
use helperText instead